src/components/ViewEditor.js

```diff
@@ -27,6 +28,7 @@ class ViewEditor extends React.Component {
 				alert('failed to retrieve view');
 				handleSetCurrentOpenFileId(NO_OPEN_FILE_ID);
 			} else {
+				this.props.setTagsInView([]); // clear any tagsInView currently stored
```

Comment:
This fixes the bug when switching views and the old tagsInView are persisted before the new ones are loaded.